bucket/lxmusic.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "2.2.2",
+    "homepage": "http://lyswhut.github.io/lx-music-doc/",
+    "description": "a free & open source music finder",
+    "license": "Apache-2.0",
+    "shortcuts": [
+        [
+            "lx-music-desktop.exe",
+            "Lx Music Desktop"
+        ]
+    ],
+    "checkver": "github",
```

Comment:
`checkver: github` means check the release page of `homepage`, and obviously it does not exist in this case, you may use following instead: 
```Json
"checkver": {
    "github": "https://github.com/lyswhut/lx-music-desktop/"
}
```